shell/platform/darwin/macos/framework/Source/FlutterOpenGLRenderer.h

```diff
@@ -0,0 +1,86 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Cocoa/Cocoa.h>
+
+#import "flutter/shell/platform/darwin/macos/framework/Headers/FlutterEngine.h"
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterView.h"
+#import "flutter/shell/platform/embedder/embedder.h"
+
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
I despise this macro; it makes important information non-local when people are reading code--especially when reviewing and context is collapsed by default--and have never used it in the macOS code. Please use explicit annotations.

Reply:
Done